packages/core/src/notifications/types.ts

```diff
@@ -119,12 +154,14 @@ export const NotificationsStateConstructor: TypeConstructor<NotificationsState>
 
 export type NotificationType = keyof Omit<NotificationsState, 'dismissed' | 'newlyReceived'>
 
+type AuthType = 'credentials' | 'builderId' | 'identityCenter' | 'unknown'
```

Comment:
nit: Its probably best to centralize these common auth values. We have similar-ish types in `connection.ts`. Maybe this line can go in to that file for now, and we can make a note to refactor later

Reply:
I have a todo here: https://github.com/aws/aws-toolkit-vscode/blob/135c3e5a73bda9c51da47eb5004b6a6d7741ed29/packages/core/src/notifications/rules.ts#L170

but it should probably be moved somewhere more common as you mentioned